### PR TITLE
fixed storing stress cmd parameters if running with user profile

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2646,22 +2646,22 @@ def add_nodes(self, count, ec2_user_data='', dc_idx=0):
...
+                self.log.exception('Failed to create node: %s', ex)
+            else:
+                self._node_index += 1
+                self.nodes += [n]
 
-            local_nodes = [n for n in self.nodes if n.dc_idx == dc_idx]
-            if len(local_nodes) - len(nodes) > 0:
-                n.is_addition = True
+                local_nodes = [n for n in self.nodes if n.dc_idx == dc_idx]
+                if len(local_nodes) - len(nodes) > 0:
```
Comment (abvgedeika) [inline sdcm/cluster.py]: len(local_nodes) > len(nodes)

**Comment by amoskong** [comment]
@larisau ClusterTester.credentials and ScyllaGCECluster.credentials are not a list object.
Before your fix, the code is sharing one credentials for all datacenters, it works in our jobs.

What's the problem you touched?

If it's reasonable to change it to list, we also need to fix tester.py to accept/parse credentials list from yaml config.

**Comment by larisau** [comment]
The fix was caused by tearDown failure on credentials.destroy() for ScyllaAWSCluster - in this case credentials is a list. I found that initial value of credentials was changed to list in setUp() (by you :). What is the reason of this - multiple DC support? Anyway, it should be the same for all the cluster types.

**Comment by amoskong** [comment]
@larisau I just rechecked the code, your fix is correct.  Thanks!

Your code append the shared GCE credentials to the list, so there is not problem for parsing credentials parameter.

**Pull request merged**